remoteappmanager/static/js/admin/vue-components/UsersView.js

```diff
@@ -1,116 +1,141 @@
   "jsapi/v1/resources",
-  "admin/vue-components/users/NewUserDialog",
-  "admin/vue-components/users/RemoveUserDialog",
-], function(Vue, resources, NewUserDialog, RemoveUserDialog) {
+  "admin/vue-components/users/NewUserDialog"
+], function(Vue, resources, NewUserDialog) {
   "use strict";
 
   return {
     components: {
       'new-user-dialog': NewUserDialog,
-      'remove-user-dialog': RemoveUserDialog
     },
     template: 
-      '<div class="row">' +
-      '  <div class="col-md-12">' +
-      '    <div class="box">' +
-      '      <div class="box-header with-border">Users</div>' +
-      '      <div class="box-body">' +
-      '        <div class="pull-right">' +
-      '          <button class="btn btn-primary createnew" @click="showNewUserDialog = true">Create New</button>' +
-      '        </div>' +
-      '        <div class="alert alert-danger" v-if="communicationError">' +
-      '          <strong>Error:</strong> {{communicationError}}' +
-      '        </div>' +
-      '        <table id="datatable" class="display dataTable">' +
-      '          <thead>' +
-      '          <tr>' +
-      '              <th>ID</th>' +
-      '              <th>Username</th>' +
-      '              <th>Accounting</th>' +
-      '              <th>Remove</th>' +
-      '          </tr>' +
-      '          </thead>' +
-      '          <tbody>' +
-      '            <tr v-for="(u, index) in users">' +
-      '              <td>{{ u.id }}</td>' +
-      '              <td>{{ u.name }}</td>' +
-      '              <td><router-link :to="{ name: '+"'"+'user_accounting'+"'"+', params: { id: u.id }}">Show</router-link></td>' +
-      '              <td><button class="btn btn-danger" @click="removeAction(index)">Remove</button></td>' +
-      '            </tr>' +
-      '          </tbody>' +
-      '        </table>' +
-      '        <new-user-dialog ' +
-      '          v-if="showNewUserDialog"' +
-      '          :show="showNewUserDialog"' +
-      '          @created="newUserCreated"' +
-      '          @closed="newUserDialogClosed"></new-user-dialog>' +
-      '        <remove-user-dialog ' +
-      '          v-if="showRemoveUserDialog"' +
-      '          :userToRemove="userToRemove"' +
-      '          @removed="userRemoved"' +
-      '          @closed="removeDialogClosed"></remove-user-dialog>' +
-      '      </div>' +
+      '<adminlte-box title="Users">' +
+      '    <div class="alert alert-danger" v-if="communicationError">' +
+      '      <strong>Error:</strong> {{communicationError}}' +
       '    </div>' +
-      '  </div>' +
-      '</div>',
+      '    <data-table' +
+      '     :headers.once="table.headers"' +
+      '     :rows="table.rows"' +
+      '     :globalActions="table.globalActions"' +
+      '     :rowActions="table.rowActions">' +
+      '    </data-table>' +
+      '    <new-user-dialog ' +
+      '      v-if="newUserDialog.show"' +
+      '      :show="newUserDialog.show"' +
```

Comment:
the new dialog needs to know if it's visible at the moment, so it needs that information. I am leaving the New dialogs unchanged for the moment, because I need a better dialog for them (and I am not sure it's worth extracting it)